scripts/install-deps.sh

```diff
@@ -43,7 +43,7 @@ use_ssh=false
 
 install_aduc_deps=false
 install_azure_iot_sdk=false
-azure_sdk_ref=LTS_07_2021_Ref01
+azure_sdk_ref=lts_08_2023
```

Comment:
Let's change it to be all caps to match the tagname:
[LTS_08_2023](https://github.com/Azure/azure-iot-sdk-c/releases/tag/LTS_08_2023)